relion/protocols/protocol_subtract.py

```diff
@@ -86,14 +122,26 @@ def _defineParams(self, form):
                            "That is: *the mask should INCLUDE the part of the "
                            "volume that you wish to KEEP.*")
 
+        form.addParam('invertMask', BooleanParam, default=False,
+                      label='Invert Mask',
+                      condition="not relionInput",
+                      help="Invert the provided mask so 0 became 1 and viceversa"
+                      )
+
         form.addSection('Centering')
+        form.addParam('help1', LabelParam,
```

Comment:
wth is this... please remove

Reply:
The problem that I tried to solve here is the following.  Scipion allows developers to show or hide a parameter using the "condition" tag. This is not the case for "tab". The new version of subtract-projection has a tab that is not needed by the old-version. What should I do with this extra tab? I just show the message "Empty section (see help)" I do not think leaving the tab empty is a better option.